filters-share/specific_ELEMHIDE.txt

```diff
@@ -1,3 +1,4 @@
+www.instiz.net##.footad
```

Comment:
Cloud not find.
However, It can be covert to `##div[class$="ad"]`.

Reply:
fix 3ad6cbb